### PR TITLE
16.0 migrate db slitheryduke11 70567

### PR DESCRIPTION
**Pull request by slitheryduke11**

### CODE REVIEW

**Comment by slitheryduke11** [comment]
Hi, @luisg123v.

I think this is ready. Could you help me, please?

File: .gitmodules [hunk 1]
```diff
@@ -1,20 +1,12 @@
...
-	branch = 12.0
+	branch = 16.0
+[submodule "costarica"]
+	path = costarica
+	url = git@git.vauxoo.com:vauxoo/costarica.git
+	branch = 16.0
+[submodule "edi-document"]
+	path = edi-document
+	url = git@git.vauxoo.com:vauxoo/edi-document.git
+	branch = 16.0
```
Comment (luisg123v) [inline .gitmodules:12]: Missing newline at end of file

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by slitheryduke11** [comment]
Hi, @luisg123v.

Is this PR alright?

**Comment by luisg123v** [comment]
Target branch is not correct.

**Comment by slitheryduke11** [comment]
> Target branch is not correct.

Done.

**Pull request merged**